terrasync/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12 as builder
+
+COPY . /src
+WORKDIR /src
+RUN CGO_ENABLED=0 go build -o terrasync .
```

Comment:
consider adding small comment on why we need to interoperate with c

Reply:
so this is explicity _not_ linking against the C libraries and creating a static executable.  It's not really necessary since these both have gcc libraries present so I can remove it